Documentation/cmdref/cilium-agent.md

```diff
@@ -22,12 +22,14 @@ cilium-agent
       --auto-ipv6-node-routes             Automatically adds IPv6 L3 routes to reach other nodes for non-overlay mode (--device) (BETA)
       --bpf-root string                   Path to BPF filesystem
       --config string                     Configuration file (default "$HOME/ciliumd.yaml")
+      --container-runtime stringSlice     Sets the container runtime(s) used by Cilium { docker | none | auto }" (default [auto])
+      --container-runtime-endpoint map    Container runtime(s) endpoint(s). (default: --container-runtime-endpoint=docker=unix:///var/run/docker.sock) (default map[])
```

Comment:
the default on this is printed twice. The second instance seems autogenerated. I don't suppose the custom option can include the default value as part of it's init?

Reply:
Oh, also, can we define multiple of these? If so, can we use just --container-runtime-endpoint to both set the path and the runtime(s) to use? instead of also having --container-runtime? (I'm expecting that unless --container-runtime=[none] the use would always have to define a 1:1 corresponding --container-runtime-endpoint option.
If you do think this is worth doing, I'd also say use --container-runtime as the name of the parameter:
```
--container-runtime=docker # use default docker endpoint
--container-runtime=docker=unix://mysock.sock --container-runtime=crio # use custom docker socket, default crio socket
--container-runtime=none # no runtime
```

Reply:
Thats fine. I assumed we didn't want to keep making more command line options (but then make one that takes a more complex set of values) but I have no strong preference in the end.